backend/.env

```diff
@@ -1 +1,2 @@
-SECRET="secret_string"
\ No newline at end of file
+SECRET="secret_string"
+MONGO_URI = "mongodb://localhost:27017/"
```

Comment:
The URI is not needed in the .env file (when committing); please remove it. It is OK to have it locally.

Reply:
Oh got it.